database/snapshotCLI.ts

```diff
@@ -1,6 +1,6 @@
 import takeSnapshot from '../src/components/snapshots/takeSnapshot'
 import useSnapshot from '../src/components/snapshots/useSnapshot'
-import { DEFAULT_SNAPSHOT_NAME, DEFAULT_OPTIONS_NAME } from '../src/components/snapshots/constants'
+import { DEFAULT_SNAPSHOT_NAME, DEFAULT_OPTIONS_NAME } from '../src/constants'
```

Comment:
I moved the "constants.ts" file out of the snapshot area, because we want to use it in more contexts than just snapshots.